keras_cv/models/backbones/backbone_presets.py

```diff
@@ -11,14 +11,17 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
 """All Backbone presets"""
 
+from keras_cv.models.backbones.mobilenet_v3 import mobilenet_v3_backbone_presets
 from keras_cv.models.backbones.resnet_v1 import resnet_v1_backbone_presets
 from keras_cv.models.backbones.resnet_v2 import resnet_v2_backbone_presets
 
 backbone_presets_no_weights = {
     **resnet_v1_backbone_presets.backbone_presets_no_weights,
     **resnet_v2_backbone_presets.backbone_presets_no_weights,
+    **mobilenet_v3_backbone_presets.backbone_presets_no_weights,
 }
 
 backbone_presets_with_weights = {
```

Comment:
Question for @jbischof -- should we include a `mobilenet_v3_backbone_presets.backbone_presets_with_weights` even though it would be empty?

Seems to me like it might be nice to include it, even as an empty dict, for consistency.